8.0/debian/buster/developer/Dockerfile

```diff
@@ -0,0 +1,70 @@
+FROM php:8.0-buster
+MAINTAINER "Bence Gazder"
```

Comment:
`MAINTAINER` is deprecated, please don't use it. (https://docs.docker.com/engine/reference/builder/#maintainer-deprecated)

The same applies to the other images as well.

Reply:
Removed it. Now only that LABEL is present (should be enough)